core/services/ocr3/plugins/ccip/commit/plugin_functions.go

```diff
@@ -244,52 +240,52 @@ func newMsgsConsensusForChain(
 	lggr.Debugw("observed messages consensus",
 		"chain", chainSel, "fChain", fChain, "observedMsgs", len(observedMsgs))
 
-	// First come to consensus about the (sequence number, id) pairs.
-	// For each sequence number consider correct the ID with the most votes.
-	msgSeqNumToIDCounts := make(map[cciptypes.SeqNum]map[string]int) // seqNum -> msgID -> count
+	// First come to consensus about the (sequence number, msg hash) pairs.
+	// For each sequence number consider correct the Hash with the most votes.
```

Comment:
Is that what you meant here?
```suggestion
	// For each sequence number consider the Hash with the most votes.
```

Reply:
I guess it's the same thing. Updating to what you recommend.